src/Vendoring/OpenTelemetry.Instrumentation.ConfluentKafka/ConfluentKafkaCommon.cs

```diff
@@ -3,7 +3,6 @@
 
```

Comment:
We shouldn't be making changes in the `Vendoring` code. We want this to be exactly what is in the upstream, to make it easier to update in the future.